src/test/unit/lambda_tracer/test_tracer.py

```diff
@@ -186,30 +185,28 @@ def lambda_test_function(event, context):
     assert Configuration.should_report == "123"
 
 
-def test_wrapping_print_happy_flow(context):
+def test_wrapping_print_happy_flow(context, capsys):
     @lumigo_tracer()
     def lambda_test_function(event, context):
         print("hello")
         return 1
 
-    with CaptureOutput() as capturer:
-        assert lambda_test_function({}, context) == 1
-        assert any(line == "hello" for line in capturer.get_lines())
+    assert lambda_test_function({}, context) == 1
+    assert "hello" in capsys.readouterr().out
 
 
-def test_wrapping_enhanced_print_backward_compatible(context):
+def test_wrapping_enhanced_print_backward_compatible(context, capsys):
     @lumigo_tracer(enhance_print=True)
     def lambda_test_function(event, context):
         print("hello")
         return 1
 
-    with CaptureOutput() as capturer:
-        assert lambda_test_function({}, context) == 1
-        assert any(line == "hello" for line in capturer.get_lines())
+    assert lambda_test_function({}, context) == 1
```

Comment:
unrelated change as well - I removed this external dependency and used a native solution. It is also faster